app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/settings/SettingsPanel.tsx

```diff
@@ -204,7 +206,7 @@ export const SettingsPanel = ({ keyId = false }: { keyId?: string | false }) =>
         </div>
       </div>
       <SettingsModal
-        keyId={keyId}
+        keyId={apiKey}
```

Comment:
Let's use apiKeyId here in place of ` keyId` and `apiKey` it's not really an `apiKey` so want to avoid the confusion,